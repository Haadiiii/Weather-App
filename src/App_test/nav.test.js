import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../Redux/Store';
import Nav from '../components/Nav';

test('testin navigation', () => {
  const nav = render(
    <React.StrictMode>
      <Provider store={store}>
        <BrowserRouter>
          <Nav />
        </BrowserRouter>
      </Provider>
    </React.StrictMode>,
  );
  expect(nav).toMatchSnapshot();
});
