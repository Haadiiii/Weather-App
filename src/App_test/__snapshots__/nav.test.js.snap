// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testin navigation 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="main"
      >
        <div
          class="nav"
        >
          <h1>
            Weather App
          </h1>
          <ul>
            <li>
              <a
                href="/"
              >
                Home
              </a>
            </li>
            <li>
              <a
                href="/contact"
              >
                Contact
              </a>
            </li>
          </ul>
        </div>
        <div>
          <input
            placeholder="Enter a city"
            type="text"
            value=""
          />
          <div
            class="sectionOne"
          >
            <div
              class="sectorOne"
            />
            <div
              class="sectorTwo"
            />
          </div>
          <div
            class="sectionTwo"
          >
            <div
              class="left"
            >
              <p>
                Date and Time
              </p>
            </div>
            <div
              class="middle"
            >
              <p>
                Timezone
              </p>
            </div>
            <div
              class="right"
            >
              <p>
                Wind Speed
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="main"
    >
      <div
        class="nav"
      >
        <h1>
          Weather App
        </h1>
        <ul>
          <li>
            <a
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/contact"
            >
              Contact
            </a>
          </li>
        </ul>
      </div>
      <div>
        <input
          placeholder="Enter a city"
          type="text"
          value=""
        />
        <div
          class="sectionOne"
        >
          <div
            class="sectorOne"
          />
          <div
            class="sectorTwo"
          />
        </div>
        <div
          class="sectionTwo"
        >
          <div
            class="left"
          >
            <p>
              Date and Time
            </p>
          </div>
          <div
            class="middle"
          >
            <p>
              Timezone
            </p>
          </div>
          <div
            class="right"
          >
            <p>
              Wind Speed
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
